src/validators/account-validator.js

```diff
@@ -0,0 +1,231 @@
+        .custom((value, { request }) => {
+            try{
+                const dateBirth = new Date(value).getFullYear();
+                const dateNow = new Date().getFullYear();
+                const year = dateNow- dateBirth;
+                if(year < 10 || year > 100 ){
+                    return false;
+                }
+                return true;
+            }catch(error){
+                return false;
+            }     
+        }).withMessage('El rango de la fecha actual y la fecha de nacimiento debe estar entre 10 y 100 años.')
+]
+
+const validationUsername = [
+    check('username')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 3, max: 20}).withMessage('El nombre de usuario debe tener un mínimo de 3 caracteres y un máximo de 20 caracteres')
+        .matches(/^[A-Za-z0-9]{3,20}$/).withMessage('El nombre de usuario debe tener solo letras y números')
+]
+
+const validationEmail = [
+    check('email')
+        .normalizeEmail()
+        .isEmail().withMessage('Debe ser un correo electrónico')
+]
+
+const validationPassword = [
+    check('password')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 8, max:254 }).trim().withMessage('Password must have a minimum of 8 characters and a maximum of 254 characters')
+]
+
+
+const validationLogin = [
+    validationUsername,
+    validationPassword,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationConfirmationAccount = [
+    validationUsername,
+    check('codeConfirmation')
+        .exists().withMessage('El campo debe existir')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .not().isString().withMessage('El campo debe ser un número')
+        .matches(/^[0-9]{6}$/).withMessage('Permitir solo números del 0 al 9 y deben ser 6 dígitos'),
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationSendEmail = [
+    validationEmail,
+    (request, response, next) => {
+        validateResult(request, response, next);
+    }
+]
+
+const validationAccountEmail = [
+    validationEmail,
+    check('title')
+        .notEmpty().withMessage('El campo no debe estar vacío')
+        .isString().withMessage('El campo debe ser una cadena')
+        .isLength({ min: 5, max: 100}).withMessage('El titulo del mensaje debe tener un mínimo de 5 caracteres y un máximo de 100 caracteres')
+        .matches(/^[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]+(\s*[a-zA-ZÑñÁáÉéÍíÓóÚúÜü0-9.,#]*){5,600}/).withMessage('Solo letras de la A a la Z, números del 0 al 9, caracteres., # y espacios'), 
+    check('message')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Martha-Miroslava/APIBiziidiDiidxaza/pullRequest?prid=8267260)
```suggestion
    check("message")
```